src/components/pages/account-overview/account-overview.tsx

```diff
@@ -26,12 +32,17 @@ const AccountOverview: React.FC = () => {
     <>
       {Object.entries(SECTION_LIST_CONTENT).map(
         ([sectionName, { text, icon }]) => {
+          const sectionHref =
+            sectionName.toLocaleLowerCase() === "personal_information"
+              ? "/account/profile"
+              : `/account/${sectionName.toLocaleLowerCase()}`;
+
           return (
```

Comment:
Need a check for feature flag env var here too, no ?

Reply:
74ee1ba